hazelcast-hibernate5/src/main/java/com/hazelcast/hibernate/local/LocalRegionCache.java

```diff
@@ -106,6 +103,14 @@ public LocalRegionCache(final String name, final HazelcastInstance hazelcastInst
         }
     }
 
+    private static MapConfig lookupMapConfig(String name, HazelcastInstance hazelcastInstance) {
+        try {
+            return hazelcastInstance != null ? hazelcastInstance.getConfig().getMapConfig(name) : null;
```

Comment:
I've changed this from using `findMapConfig` to using `getMapConfig`, which has fewer weird side-effects but should still give us what we need.